v4-client-js/examples/gov_add_new_market.ts

```diff
@@ -1,16 +1,13 @@
-import { EncodeObject, Registry } from '@cosmjs/proto-signing';
 import Long from 'long';
 
-import { BECH32_PREFIX } from '../src';
+import { GovAddNewMarketParams, LocalWallet, ProposalStatus } from '../src';
 import { CompositeClient } from '../src/clients/composite-client';
-import { Network } from '../src/clients/constants';
-import { generateRegistry } from '../src/clients/lib/registry';
-import { Composer } from '../src/clients/modules/composer';
-import LocalWallet from '../src/clients/modules/local-wallet';
+import { BECH32_PREFIX, Network } from '../src/clients/constants';
+import { sleep } from '../src/lib/utils';
 import { DYDX_LOCAL_MNEMONIC } from './constants';
 
 const INITIAL_DEPOSIT_AMOUNT = 10_000_000_000_000; // 10,000 whole native tokens.
```

Comment:
Could we export `INITIAL_DEPOSIT_AMOUNT`, `getTitle`, `getSummary` as part of a newMarketUtils export? That way if values like `INITIAL_DEPOSIT_AMOUNT` change, it'll only need to be changed in one place.

Reply:
if we could also export `DEFAULT_DELAY_BLOCKS` as well

Reply:
For `INITIAL_DEPOSIT_AMOUNT`, this value is different across different networks. So if we add a default amount here, you'd likely need to change it based on which network you are talking to.

For `DEFAULT_DELAY_BLOCK`, fine for the client to have a default value but note that for testnet testing, this value should be smaller so we can see the effects faster. Otherwise, we'd need to wait another day or so.

For `getTitle` and `getSummary`, makes sense

Reply:
okay! sounds good

Reply:
Discussed offline. Decided not to add these default values in the client. Web will introduce these as env vars.